<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="generateEntries">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Generate entries</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="typeSelectLabel">
       <property name="text">
        <string>group type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeSelectBox">
       <item>
        <property name="text">
         <string>prim</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>point</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>edge</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>vertex</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="renameButton">
       <property name="text">
        <string>Rename!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentNodeButton">
       <property name="text">
        <string>CURRENT NODE: None</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Screenshot geomtry shading mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="shadingModeComboBox">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Use wireframe ghost</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Keep viewport shading mode</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>590</width>
        <height>254</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
